Source/WebCore/rendering/style/RenderStyleSetters.h

```diff
@@ -148,9 +144,9 @@ inline void RenderStyle::setColumnRuleWidth(unsigned short w) { SET_DOUBLY_NESTE
 inline void RenderStyle::setColumnSpan(ColumnSpan span) { SET_DOUBLY_NESTED(m_nonInheritedData, miscData, multiCol, columnSpan, static_cast<unsigned>(span)); }
 inline void RenderStyle::setColumnWidth(float width) { SET_DOUBLY_NESTED_PAIR(m_nonInheritedData, miscData, multiCol, width, width, autoWidth, false); }
 inline void RenderStyle::setContain(OptionSet<Containment> containment) { SET_NESTED(m_nonInheritedData, rareData, contain, containment); }
-inline void RenderStyle::setContainIntrinsicHeight(std::optional<Length> height) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicHeight, height); }
+inline void RenderStyle::setContainIntrinsicHeightLength(std::optional<Style::Length<CSS::Nonnegative, float>> height) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicHeight, height); }
 inline void RenderStyle::setContainIntrinsicHeightType(ContainIntrinsicSizeType containIntrinsicHeightType) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicHeightType, static_cast<unsigned>(containIntrinsicHeightType)); }
-inline void RenderStyle::setContainIntrinsicWidth(std::optional<Length> width) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicWidth, width); }
+inline void RenderStyle::setContainIntrinsicWidthLength(std::optional<Style::Length<CSS::Nonnegative, float>> width) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicWidth, width); }
 inline void RenderStyle::setContainIntrinsicWidthType(ContainIntrinsicSizeType containIntrinsicWidthType) { SET_NESTED(m_nonInheritedData, rareData, containIntrinsicWidthType, static_cast<unsigned>(containIntrinsicWidthType)); }
```

Comment:
Doesn't need to be done here, but it would be cool if each of those just used one field, rather than 2 each. Maybe this needs a `CSS::ContainIntrinsicSize` structure or such.